docs/source/results.rst

```diff
@@ -45,6 +45,11 @@ Below is a comparison of some vectors shown in 2D compared to 3D.
  
 The arrows are coloured according to their relative size. Red arrows are the largest and dark blue the smallest. Note that the colours of the arrows may differ between the 2D and 3D viewer because the colouring of the 2D arrows is only taking into account the size of the displacements in two, rather than all three directions. 
```

Comment:
Colours need updating:

```suggestion
The arrows are coloured according to their relative size. White arrows are the largest and black the smallest. Note that the colours of the arrows may differ between the 2D and 3D viewer because the colouring of the 2D arrows is only taking into account the size of the displacements in two, rather than all three directions. 
```